### DISCUSSION TITLE
Do not play jingles

### DISCUSSION THREAD

**Discussion by cheleentano** [savonet/liquidsoap#1763] [Q&A]
Hi everyone, I apologize for my poor English. I want to set up the jingles to play at the beginning of every hour. My config, but it does not work. Help fix for the current version? Liquidsoap 1.4.1
```
music = mksafe(playlist(mode="randomize", reload=1, reload_mode="rounds", "/home/music"))
jingles = mksafe(playlist("/home/music/jingles"))

radio = crossfade(fade_out = 6.0, fade_in = 6.0, music)
radio = random(weights=[1,10],[ jingles, radio ])
```

**Comment by smimram** [comment]
What do you mean that it does not work? Your script should play a jingle every 10 songs (roughly). Isn't this the case? Many strategies for jingles are listed in [the book](https://www.liquidsoap.info/doc-dev/book.html).
